tests/integration/fileTypeTests.py

```diff
@@ -29,57 +30,25 @@ def setUpClass(cls):
         print("Uploading files")
         # Upload needed files to S3
         s3FileNameValid = cls.uploadFile("appropValid.csv", user)
-        s3FileNameMixed = cls.uploadFile("appropMixed.csv", user)
         s3FileNameProgramValid = cls.uploadFile("programActivityValid.csv", user)
-        s3FileNameProgramMixed = cls.uploadFile("programActivityMixed.csv", user)
         s3FileNameAwardFinValid = cls.uploadFile("awardFinancialValid.csv", user)
-        s3FileNameAwardFinMixed = cls.uploadFile("awardFinancialMixed.csv", user)
         s3FileNameAwardValid = cls.uploadFile("awardValid.csv", user)
-        s3FileNameAwardMixed = cls.uploadFile("awardMixed.csv", user)
-        s3FileNameAwardMixedDelimiter = cls.uploadFile("awardMixedDelimiter.csv", user)
-        s3FileNameCrossAwardFin = cls.uploadFile("cross_file_C.csv", user)
-        s3FileNameCrossAward = cls.uploadFile("cross_file_D2.csv", user)
-        s3FileNameCrossApprop = cls.uploadFile("cross_file_A.csv", user)
-        s3FileNameCrossPgmAct = cls.uploadFile("cross_file_B.csv", user)
-        s3FileNameAppropValidShortcols = cls.uploadFile("appropValidShortcols.csv", user)
-        s3FileNameProgramMixedShortcols = cls.uploadFile("programActivityMixedShortcols.csv", user)
-        s3FileNameAwardFinMixedShortcols = cls.uploadFile("awardFinancialMixedShortcols.csv", user)
-        s3FileNameAwardValidShortcols = cls.uploadFile("awardValidShortcols.csv", user)
 
         # Create submissions and get IDs back
         submissionIDs = {}
-        for i in range(0, 16):
-            if i == 7:
-               # Award financial mixed will be second quarter
-                submissionIDs[i] = cls.insertSubmission(cls.jobTracker, user, datetime(2015,3,15))
-            else:
-                submissionIDs[i] = cls.insertSubmission(cls.jobTracker, user)
+        for i in range(0, 9):
```

Comment:
So the reason the submission numbers are done this way is we don't want to have conflicting submission IDs between this and mixedFileTests.  They're in separate DBs, but the reports get written to the same place, so those would be overwritten if they used the same submission ID.  I could rearrange the numbers so that this is 1-4 and mixed is 5-15.